Assignments/Task-2/MiddleCharacter.txt

```diff
@@ -0,0 +1,15 @@
+//https://www.codewars.com/kata/56747fd5cb988479af000028/train/javascript
+
+
+function getMiddle(s)
+{
+  var result;
+  var length = s.length;
+  var middle = length/2;
+  if(length % 2 == 0){
+      result = s.substr(middle-1, 2);
```

Comment:
substr is not a standard method, please update this with substring method

Reply:
code updated with the suggested changes